.github/workflows/release.yml

```diff
@@ -2,39 +2,40 @@ name: release
 
 on:
   schedule:
-    - cron: '0 0 * * SAT' # every Saturday
+    - cron: "0 0 * * SAT" # every Saturday
   workflow_dispatch:
 
 jobs:
   build:
-     runs-on: ubuntu-18.04
+    runs-on: ubuntu-18.04
 
-     env:
-       OPAMJOBS: 2
-       OPAMRETRES: 8
-       VERSION: 2.3.0
+    env:
+      OPAMJOBS: 2
+      OPAMRETRES: 8
+      VERSION: 2.3.0
 
-     steps:
-         - name: Checkout code
-           uses: actions/checkout@v2
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
 
-         - name: Install OCaml
-           uses: ocaml/setup-ocaml@v2
-           with:
-             ocaml-compiler: 4.11.1+flambda
+      - name: Install OCaml
+        uses: ocaml/setup-ocaml@v2
+        with:
+          ocaml-compiler: ocaml-variants.4.11.2+flambda
```

Comment:
There are quite a few changes, but the only thing to keep in mind is here. Since v2, we need the full package name, so we need to do this.